pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,26 @@
+{stdenv, fetchurl, which, go-md2man}:
+
+let
+  v = "1.4.3";
+in
+stdenv.mkDerivation {
+  version = v;
+  name = "brillo-${v}";
+  src = fetchurl {
+  url = "https://gitlab.com/cameronnemo/brillo/-/archive/v${v}/brillo-v${v}.tar.bz2";
+    sha256 = "0wjpw9vn521rwpnlhvczzfzdsdq812vsyl151627qw51zadynvz1";
+  };
+  patches = [./brillo.patch];
```

Comment:
Can you briefly explain the rationale for the patch either in the patch header or in a comment here?
Installing apparmor rules seems harmless to me; NixOS even has (some) apparmor support.

Reply:
I've just made another commit which changes all this

Reply:
should I add something along the lines of security.apparmor.packages = [pkgs.brillo] ?